hyperjumptech/monika
# feat(config): create default config file when there is none specified

# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1. Create default configuration file if not defined in the first place

## How did you implement / how did you fix it  
Create monika.yml file in the root directory where monika is run, the definition is taken from the configuration file  

## How to test  
1. delete all monika configurations
2. run monika without any config : `npm run start`
3. new `monika.yml` config should be created in folder where the monika run
